app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TweetController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
```

Comment:
we should keep with the design patterns for CRUD with MVC and have set names for these files 

- index.erb
- show.erb
- new.erb
- edit.erb